src/cljx/racehub/util/time.cljx

```diff
@@ -154,14 +166,24 @@
    (time/interval (time/epoch)
                   (timestamp-to-datetime ts))))
 
+(s/defn unix-time :- UnixTime
```

Comment:
`(with-time blah (= (unix-time->datetime (unix-time)) (now))`, something like that